database/engine/README.md

```diff
@@ -43,51 +43,48 @@ actual page cache size will be slightly larger than this figure—see the [memor
+Netdata metric values per legacy database engine instance (see [details on the legacy mode](#legacy-mode) below).
 
 ```conf
 [global]
     dbengine disk space = 256
 ```
 
-Use the  [**database engine calculator**](https://learn.netdata.cloud/docs/agent/database/calculator) to correctly set
-`dbengine disk space`(**deprecated**) based on your needs. The calculator gives an accurate estimate based on how many
-child nodes you have, how many metrics your Agent collects, and more.
-
 ### Streaming metrics to the database engine
 
-##### Legacy mode
-
-When streaming metrics, the Agent on the parent node used to create one instance (legacy, version <= 1.23.2) of the
-database engine for itself, and another instance for every child node it receives metrics from. If you had four
-streaming nodes, you would have five instances in total (`1 parent + 4 child nodes = 5 instances`).
+When using the multihost database engine, all parent and child nodes share the same `page cache size` and `dbengine
+multihost disk space` in a single dbengine instance. The [**database engine
+calculator**](https://learn.netdata.cloud/docs/agent/database/calculator) helps you properly set `page cache size` and
+`dbengine multihost disk space` on your parent node to allocate enough resources based on your metrics retention policy
+and how many child nodes you have.
 
-The Agent allocated resources for each instance separately using the `dbengine disk space`(**deprecated**) setting. If
-`dbengine disk space`(**deprecated**) is set to the default `256`, each instance is given 256 MiB in disk space, which
-means the total disk space required to store all instances is, roughly, `256 MiB * 1 parent * 4 child nodes = 1280 MiB`. 
+#### Legacy mode
 
-See the [database engine calculator](https://learn.netdata.cloud/docs/agent/database/calculator) to help you correctly
-set `dbengine disk space`(**deprecated**) and understand the total disk space required based on your streaming setup.
+_For Netdata Agents earlier than v1.23.2_, the Agent on the parent node uses one dbengine instance for itself, and
+another instance for every child node it receives metrics from. If you had four streaming nodes, you would have five
+instances in total (`1 parent + 4 child nodes = 5 instances`).
 
-##### Multi host DB mode
-
-In the newer agent versions the parent and child nodes all share `page cache size` and `dbengine multihost disk space`
-in a single dbengine multi-host instance.
+The Agent allocates resources for each instance separately using the `dbengine disk space` (**deprecated**) setting. If
+`dbengine disk space`(**deprecated**) is set to the default `256`, each instance is given 256 MiB in disk space, which
+means the total disk space required to store all instances is, roughly, `256 MiB * 1 parent * 4 child nodes = 1280 MiB`.
 
-##### Backward compatibility
+#### Backwards compatibility
```

Comment:
In my defense wikipedia says [backward](https://en.wikipedia.org/wiki/Backward_compatibility)!

Reply:
Backward it is then! :)